<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="handGestureButton">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>200</y>
      <width>400</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Hand Gesture Recognition</string>
    </property>
   </widget>
   <widget class="QPushButton" name="facialExpressionButton">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>300</y>
      <width>400</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Facial Expression Recognition</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colourButton">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>400</y>
      <width>400</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Colour Detection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="objectButton">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>500</y>
      <width>400</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Object Detection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lidarButton">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>600</y>
      <width>400</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>LiDAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="thermalButton">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>700</y>
      <width>400</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Thermal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eventButton">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>800</y>
      <width>400</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Event</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>620</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Welcome</string>
    </property>
   </widget>
   <widget class="QLabel" name="description">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>1100</width>
      <height>470</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>This is a multi-sensor demonstration application that integrates an RGB camera, Event camera, LiDAR and Thermal camera to showcase their functionality and unique capabilities. It also includes recognition algorithms for the RGB camera, allowing for real-time detection and analysis of objects, gestures and facial expressions.

Click on a button to start exploring and enjoy the learning experience!
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>850</y>
      <width>761</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QLabel" name="connectionLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>360</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Connection Status</string>
     </property>
    </widget>
    <widget class="QLabel" name="rgbLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <fontweight>Medium</fontweight>
      </font>
     </property>
     <property name="text">
      <string>RGB</string>
     </property>
    </widget>
    <widget class="QLabel" name="rgbCircle">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lidarLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <fontweight>Medium</fontweight>
      </font>
     </property>
     <property name="text">
      <string>LiDAR</string>
     </property>
    </widget>
    <widget class="QLabel" name="lidarCircle">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="thermalLabel">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>60</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <fontweight>Medium</fontweight>
      </font>
     </property>
     <property name="text">
      <string>Thermal</string>
     </property>
    </widget>
    <widget class="QLabel" name="thermalCircle">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>60</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="eventLabel">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>60</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <fontweight>Medium</fontweight>
      </font>
     </property>
     <property name="text">
      <string>Event</string>
     </property>
    </widget>
    <widget class="QLabel" name="eventCircle">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>60</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>824</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Datasets/UGLogo.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="QR_Code">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>690</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Datasets/QRcodes/home_QR.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="QR_label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>690</y>
      <width>970</width>
      <height>130</height>
     </rect>
    </property>
    <property name="font">
      <font>
       <pointsize>16</pointsize>
       <fontweight>Medium</fontweight>
      </font>
     </property>
    <property name="text">
     <string>Scan this QR code to visit the sensor rig website and learn more about the sensors!</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
